packaging/makeself/install-or-update.sh

```diff
@@ -212,9 +212,9 @@ run chown -R ${NETDATA_USER}:${NETDATA_GROUP} /opt/netdata
 
 # -----------------------------------------------------------------------------
 
-progress "fix plugin permissions"
+progress "changing plugins ownership and setting setuid"
 
-for x in apps.plugin freeipmi.plugin ioping cgroup-network ebpf.plugin; do
+for x in apps.plugin freeipmi.plugin ioping cgroup-network ebpf.plugin perf.plugin slabinfo.plugin freeipmi.plugin nfacct.plugin xenstat.plugin; do
```

Comment:
```suggestion
for x in apps.plugin freeipmi.plugin ioping cgroup-network ebpf.plugin perf.plugin slabinfo.plugin nfacct.plugin xenstat.plugin; do
```

"freeipmi.plugin" is 2nd item in the list.

Also, "perf.plugin" and "slabinfo.plugin" don't require setuid, setting capabilities will do (`setcap`):
 - "per.plugin": cap_perfmon=ep (or cap_sys_admin depends on the kernel).
 - "slabinfo.plugin": cap_dac_read_search=ep.

But setuid will work too.

Reply:
yes, use a setcap will be a good choice, but in our env we use systemd to control capabilities.

Reply:
We have those capabilities in our [systemd until file](https://github.com/netdata/netdata/blob/9330f53a5cf0ea0106766731b981d1cb81fc1adb/system/netdata.service.in#L46-L66). Is that what you mean (CapabilityBoundingSet)?